scripts/i3-workspace-groups

```diff
@@ -275,6 +284,53 @@ def serve(i3_connection, server_addr):
             connection.close()
 
 
+def get_monitor_active_group(controller, groups_to_workspaces, monitor):
+    active_group = ''
+    min_global = float('inf')
+    for (group, workspaces) in groups_to_workspaces.items():
+        for ws in workspaces:
+            if not monitor or _get_workspace_field(controller, ws,
+                                                   'monitor') == monitor:
+                global_number = _get_workspace_field(controller, ws,
+                                                     'global_number')
+                if global_number and global_number < min_global:
+                    active_group = group
+                    min_global = global_number
+    return active_group
+
+
+def _print_polybar_hook(controller, args):
+    workspaces = controller.get_tree().workspaces()
+    group_to_workspaces = workspace_names.get_group_to_workspaces(workspaces)
+    active_group = get_monitor_active_group(controller, group_to_workspaces,
+                                            args.monitor)
+    res = []
+    for group in sorted(group_to_workspaces.keys()):
+        workspaces = group_to_workspaces[group]
+
+        parsed_names_dict = {}
+        for ws in workspaces:
+            if not args.monitor or _get_workspace_field(
+                    controller, ws, 'monitor') == args.monitor:
+                local_number = _get_workspace_field(controller, ws,
+                                                    'local_number')
+                focused = _get_workspace_field(controller, ws, 'focused')
+                parsed_names_dict[local_number] = {'focused': focused}
+
+        if len(parsed_names_dict) != 0:
+            parsed_names = ''.join([
+                f'%{{u{args.line_color}}}%{{+u}} {local_number} %{{-u}}'
+                if parsed_names_dict[local_number]['focused'] else
+                f' {local_number} '
+                for local_number in sorted(parsed_names_dict.keys())
+            ])
+
+            formatted_group = f'%{{o{args.line_color}}}%{{+o}}{group}:%{{-o}}' if active_group == group else f'{group}:'
```

Comment:
Please wrap this line (try to limit to 80 chars which is the current convention in the code)

Reply:
I updated the code to wrap this line 👍